pkg/apis/pipeline/v1/pipeline_validation_test.go

```diff
@@ -87,7 +87,36 @@ func TestPipeline_Validate_Success(t *testing.T) {
 				Tasks: []PipelineTask{{Name: "foo", TaskRef: &TaskRef{Name: "bar", Kind: ClusterTaskRefKind}}},
 			},
 		},
-	}}
+	}, {
+		name: "valid task with pipelineRef",
+		wc:   withAlphaAPIContext,
+		p: &Pipeline{
+			ObjectMeta: metav1.ObjectMeta{Name: "pipeline"},
+			Spec: PipelineSpec{
+				Tasks: []PipelineTask{
+					{
+						Name:        "foo",
+						PipelineRef: &PipelineRef{Name: "foo-pipeline"},
+					},
+				},
+			},
+		},
+	}, {
+		name: "valid task with pipelineSpec",
+		wc:   withAlphaAPIContext,
+		p: &Pipeline{
+			ObjectMeta: metav1.ObjectMeta{Name: "pipeline"},
+			Spec: PipelineSpec{
+				Tasks: []PipelineTask{
+					{
+						Name:         "foo",
+						PipelineSpec: &PipelineSpec{Description: "foo-pipeline-description"},
+					},
+				},
+			},
+		},
+	},
+	}
```

Comment:
nit: could you format the space to make it consistent with the rest of the file?

Reply:
Thanks @QuanZhang-William Done!